benchpress/install_nginx.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -e
+set -x
+
```

Comment:
Add prerequisites here. Libraries we need to install on our test hosts before we can run this script.